go/vt/vtgate/engine/fk_verify.go

```diff
@@ -84,15 +84,19 @@ func (f *FkVerify) TryExecute(ctx context.Context, vcursor VCursor, bindVars map
 // TryStreamExecute implements the Primitive interface
 func (f *FkVerify) TryStreamExecute(ctx context.Context, vcursor VCursor, bindVars map[string]*querypb.BindVariable, wantfields bool, callback func(*sqltypes.Result) error) error {
 	for _, v := range f.Verify {
```

Comment:
Do we want a Streaming version of `fk_verify`?

Reply:
Removed 👍